common/src/main/java/net/whimxiqal/journey/data/version/SqliteDataVersionHandler.java

```diff
@@ -0,0 +1,103 @@
+package net.whimxiqal.journey.data.version;
+
+import net.whimxiqal.journey.Journey;
+import net.whimxiqal.journey.data.DataVersion;
+import net.whimxiqal.journey.data.sql.sqlite.SqliteConnectionController;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static net.whimxiqal.journey.data.DataManagerImpl.VERSION_FILE_NAME;
+import static net.whimxiqal.journey.data.DataManagerImpl.VERSION_TABLE_NAME;
+import static net.whimxiqal.journey.data.DataManagerImpl.VERSION_COLUMN_NAME;
+
+public class SqliteDataVersionHandler extends SqlDataVersionHandler {
+    public SqliteDataVersionHandler(SqliteConnectionController controller) {
+        super(controller);
+    }
+
+    @Override
+    public DataVersion version() {
+        // If the version file exists we know it's on version 1.
+        if (versionFileExists()) return DataVersion.V001;
```

Comment:
I think I'd prefer if this check happened after the check to the database, because someone may accidentally put the file back or something else that's in the realm of possibility, and the existence of the version table should take precedence over the file existing